benchmark/src/main/java/com/example/benchmark/Utils.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.benchmark
+
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+
+const val JCA_PACKAGE = "com.google.jetpackcamera"
+const val DEFAULT_ITERATIONS = 5
```

Comment:
Lets specify what iterations these are

Reply:
changed the val name to `DEFAULT_TEST_ITERATIONS`

let me know if you think that's sufficient